src/lib/containers/widgets/facet-nav/FacetNav.tsx

```diff
@@ -163,9 +165,11 @@ const FacetNav: React.FunctionComponent<FacetNavProps> = ({
     )
   }
 
-  const hasSelectedFacets =
-    lastQueryRequest?.query.selectedFacets !== undefined &&
-    lastQueryRequest.query.selectedFacets.length > 0
+  const hasFacetsOrFilters =
+    (lastQueryRequest?.query.selectedFacets !== undefined &&
+      lastQueryRequest.query.selectedFacets.length > 0) ||
+    (lastQueryRequest?.query.additionalFilters !== undefined &&
+      lastQueryRequest?.query.additionalFilters.length > 0)
```

Comment:
We weren't considering filters when rendering this text or the "Clear All" button.